site/src/theme/shadows.ts

```diff
@@ -0,0 +1,12 @@
+import { ThemeOptions } from "@mui/material";
+
+/** @todo see if it's possible to use keys "sm" | "lg" instead of the array index */
+
+export const shadows: ThemeOptions["shadows"] = [
+  "none",
+  "0px 2px 8px rgba(39, 50, 86, 0.04), 0px 2.59259px 6.44213px rgba(39, 50, 86, 0.06), 0px 0.5px 1px rgba(39, 50, 86, 0.10)",
+  "0px 9px 26px rgba(61, 78, 133, 0.03), 0px 7.12963px 18.37px rgba(61, 78, 133, 0.04), 0px 4.23704px 8.1px rgba(61, 78, 133, 0.05), 0px 0.203704px 0.62963px rgba(61, 78, 133, 0.06)",
+  "0px 20px 41px rgba(61, 78, 133, 0.04), 0px 16px 25px rgba(61, 78, 133, 0.03), 0px 12px 12px rgba(61, 78, 133, 0.02), 0px 2px 3.13px rgba(61, 78, 133, 0.01)",
+  "0px 51px 87px rgba(50, 65, 111, 0.07), 0px 33.0556px 50.9514px rgba(50, 65, 111, 0.0531481), 0px 19.6444px 27.7111px rgba(50, 65, 111, 0.0425185), 0px 10.2px 14.1375px rgba(50, 65, 111, 0.035), 0px 4.15556px 7.08889px rgba(50, 65, 111, 0.0274815), 0px 0.944444px 3.42361px rgba(50, 65, 111, 0.0168519)",
+  "0px 96px 129px rgba(61, 78, 133, 0.13), 0px 48.6px 56.2359px rgba(61, 78, 133, 0.08775), 0px 19.2px 20.9625px rgba(61, 78, 133, 0.065), 0px 4.2px 7.45781px rgba(61, 78, 133, 0.04225)",
+] as any;
```

Comment:
Why do we need to use `as any` here? This is potentially dangerous, especially if there are breaking changes in MUI later on. If we can’t add a proper typing, it is a bit safer to use `// @ts-expect-error -- explanation` instead.

Reply:
This worked for me locally:

```ts
export const shadows: ThemeOptions["shadows"] = [
  "none",
  "0px 2px 8px rgba(39, 50, 86, 0.04), 0px 2.59259px 6.44213px rgba(39, 50, 86, 0.06), 0px 0.5px 1px rgba(39, 50, 86, 0.10)",
  "0px 9px 26px rgba(61, 78, 133, 0.03), 0px 7.12963px 18.37px rgba(61, 78, 133, 0.04), 0px 4.23704px 8.1px rgba(61, 78, 133, 0.05), 0px 0.203704px 0.62963px rgba(61, 78, 133, 0.06)",
  "0px 20px 41px rgba(61, 78, 133, 0.04), 0px 16px 25px rgba(61, 78, 133, 0.03), 0px 12px 12px rgba(61, 78, 133, 0.02), 0px 2px 3.13px rgba(61, 78, 133, 0.01)",
  "0px 51px 87px rgba(50, 65, 111, 0.07), 0px 33.0556px 50.9514px rgba(50, 65, 111, 0.0531481), 0px 19.6444px 27.7111px rgba(50, 65, 111, 0.0425185), 0px 10.2px 14.1375px rgba(50, 65, 111, 0.035), 0px 4.15556px 7.08889px rgba(50, 65, 111, 0.0274815), 0px 0.944444px 3.42361px rgba(50, 65, 111, 0.0168519)",
  "0px 96px 129px rgba(61, 78, 133, 0.13), 0px 48.6px 56.2359px rgba(61, 78, 133, 0.08775), 0px 19.2px 20.9625px rgba(61, 78, 133, 0.065), 0px 4.2px 7.45781px rgba(61, 78, 133, 0.04225)",
  ...defaultTheme.shadows.slice(6),
] as ThemeOptions["shadows"]; // MUI expects to have exactly 26 shadows [... more explanation text ...]
```

Using `as` makes the compiler a bit less strict, so the chance of runtime errors increases a bit. I guess it’s fine to use typecasting here, but worth adding a comment.

Reply:
@benwerner01 you are correct. Thanks for the suggestion @kachkaev, I think that works although I'm concerned anyone might mistakenly use any of shadows 7 ... 26 which aren't in our design system. To avoid that, we should make shadows 7 - 26 the same as shadow 6. That way, if someone mistakenly tries to use shadow 7 or 8 instead of 6, it still renders the right shadow

Reply:
@teenoh sure, I've repeated the final shadow 20 times instead of deferring to the default values, and @kachkaev I've followed your suggestion to improve the type safety. Here's the commit: https://github.com/blockprotocol/blockprotocol/pull/203/commits/977725688f05593bac0be118beb325399462283d